includes/reader-revenue/class-stripe-connection.php

```diff
@@ -960,7 +960,7 @@ public static function handle_donation( $config ) {
 						'metadata' => $payment_metadata,
 					]
 				);
-				if ( ! Reader_Revenue_Emails::can_send_email( Reader_Revenue_Emails::EMAIL_TYPE_RECEIPT ) ) {
+				if ( ! Emails::can_send_email( 'receipt' ) ) {
```

Comment:
Nit: Maybe `receipt` can remain a constant in case we need to change this or add other types of receipts in the future?

Reply:
Done in 062709e97634372d7e48729465fe1ed4007009af